python/ray/_raylet.pyx

```diff
@@ -13,6 +13,7 @@ import logging
 import msgpack
 import os
 import pickle
+import pyarrow as pa
```

Comment:
I believe pyarrow is an optional dependency. You may need to move the import to where it is actually used.